zipkin-local.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
```

Comment:
Out of curiosity, why are we setting this here?

Reply:
this was also from the original `zipkin-local.js` file. One point to note is that although the original file was compiled from Typescript, this one will remain static. Gatsby only requires [a `create()` and `stop()` function](https://www.gatsbyjs.com/docs/performance-tracing/#2-library-configuration-file) for tracing, so i think with testing all these suggestions should not impact the functionality of the tracer